index.html

```diff
@@ -7,27 +7,40 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Figma Project</title>
   <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
 
 <body>
-  <div class="first_page">
-    <header class="header flex_container">
-      <div class="logo">ReMotely</div>
-      <nav class="nav flex_container">
-        <div class="flex_container"><a href="#">book</a><a href="#">explore</a><a href="#">about</a></div>
-        <div><a href="#">sign in</a></div>
-      </nav>
-    </header>
 
 
+  <header class="header flex_container">
+    <div class="logo">ReMotely</div>
+    <nav class="nav flex_container">
+      <div class="link_flex_container"><a class="links" href="#">book</a><a class="links" href="#">explore</a><a
+          class="links" href="#">about</a></div>
+      <div class="sign_container"><a class="sign" href="#">sign in</a></div>
+    </nav>
+  </header>
+
+  <section class="first_page">
+    <div class="column phrase">
     <div class="catch_phrase">
-      <h1>Work from a Cabin near Helsinki</h1>
-      <button class="btn" type="submit">Explore</button>
+      <h1>Work from <br> a Cabin near Helsinki</h1>
+    </div></div>
+    <div class="column cottage">
+    <div class="photo">
       <img class="main_photo" src="switzphoto.jpg" alt="chatka ze Szwajcarii">
-    </div>
+  </section></div>
   </div>
+  <button class="btn" type="submit">Explore</button>
```

Comment:
put this button below `<div class="section-wrapper">` closing tag and before `</section>`